posts/octavio-luna/Elasticsearch:-A-Powerful-Search-Engine-for-Backend-Engineers.md

```diff
@@ -0,0 +1,400 @@
+---
+title: 'Elasticsearch: A Powerful Search Engine for Backend Engineers'
+published: false
+description: 'Elasticsearch is a powerful search engine that can be used to search for text in documents, but it can also be used to search for structured data in JSON format. In this article, we will explore how Elasticsearch works and how it can be used to search for structured data.'
+tags: 'elasticsearch, backend, search, optimization'
+cover_image: ./assets/Elasticsearch/Elasticsearch_logo.svg.png
```

Comment:
Remove the double extension, renaming the file to ./assets/Elasticsearch/Elasticsearch_logo.png as well as physically renaming it in the /assets/Elasticsearch directory